app/assets/javascripts/backbone/views/loan_charts_view.coffee

```diff
@@ -0,0 +1,166 @@
+class MS.Views.LoanChartsView extends Backbone.View
+
+  el: '.summary-chart'
+
+  initialize: (params) ->
+    @breakevenData = params.breakeven_data
+    @breakevenFixedCosts = @breakevenData["fixed_costs"]
+    @breakevenProductionCosts = @breakevenData["cogs"]
+    @breakevenRevenue = @breakevenData["revenue"]
+    @defaultChartOptions = {
+      width: "100%"
+    }
+
+    @loadCharts()
+
+  breakevenFixedCostsChart: () ->
+    chartData = {}
+    columns = [
+      {"id":"","label":I18n.t('loan.breakeven.fixed_costs', count: 1),"pattern":"","type":"string"},
+      {"id":"","label":I18n.t('loan.breakeven.amount'),"pattern":"","type":"number"}
+    ]
+
+    rows = []
+    for key,cost of @breakevenFixedCosts
+      name = cost.name
+      total = cost.amount
+      rows.push({"c":[{"v": name, "f":null},{"v": total, "f":null}]})
```

Comment:
I was not able to convert the displayed amount, which is f, to use $ because of time constraints.